internal/providers/providers.go

```diff
@@ -210,7 +202,7 @@ func (pb *ProviderBuilder) GetGitHub() (provinfv1.GitHub, error) {
 		return nil, fmt.Errorf("error parsing github app config: %w", err)
 	}
 
-	cli, err := githubapp.NewGitHubAppProvider(cfg, pb.metrics, pb.restClientCache, gitHubCredential)
+	cli, err := githubapp.NewGitHubAppProvider(cfg, pb.cfg.GitHubApp, pb.metrics, pb.restClientCache, gitHubCredential)
```

Comment:
It seems weird to pass both the config from the definition and from the `config.yaml`. If we go down this route, should we also move the endpoint to the `config.yaml` and remove it from the provider definition?

Reply:
I'm not sure about the endpoint, could we have a minder instance that has tenants that use a private installation and other that point to the public github endpoint?

Reply:
I think that's a valid scenario. I also think we don't need to support it at this moment.